src/main/java/calculator/util/Validator.java

```diff
@@ -0,0 +1,93 @@
+package calculator.util;
+
+import static calculator.message.Delimiter.CUSTOM_DELIMITER;
+import static calculator.message.ErrorMessage.CUSTOM_DELIMITER_CANNOT_BE_NUMBER_MESSAGE;
+import static calculator.message.ErrorMessage.EMPTY_ELEMENT_MESSAGE;
+import static calculator.message.ErrorMessage.EMPTY_INPUT_MESSAGE;
+import static calculator.message.ErrorMessage.INVALID_ELEMENT_MESSAGE;
+import static calculator.message.ErrorMessage.INVALID_FLOAT_INPUT_MESSAGE;
+import static calculator.message.ErrorMessage.NEGATIVE_NUMBER_MESSAGE;
+
+import java.util.List;
+
+public class Validator {
+    public static String validateInput(String userInput) {
+        isBlank(userInput);
+        String newInput = validateCustomDelimiter(userInput);
+
+        return newInput;
+    }
+
+    public static void isNegativeNumber(List<Integer> numbers) {
+        numbers.forEach(number -> {
+            if (number < 0) {
+                throw new IllegalArgumentException(NEGATIVE_NUMBER_MESSAGE);
+            }
+        });
+    }
+
+    public static String validateCustomDelimiter(String str) {
+        if (str.length() >= 5 && str.startsWith("//") && str.charAt(3) == '\\' && str.charAt(4) == 'n') {
+            setCustomDelimiter(str);
+            validateCustomDelimiterIsNotNumber();
+            return str.substring(5);
+        }
+
+        return str;
+    }
+
+    public static void validateElement(String element) {
+        if (isBlankBetweenDelimiters(element)) {
+            throw new IllegalArgumentException(EMPTY_ELEMENT_MESSAGE);
+        }
+
+        if (isFloat(element)) {
+            throw new IllegalArgumentException(INVALID_FLOAT_INPUT_MESSAGE);
+        }
+
+        if (!isInteger(element)) {
+            throw new IllegalArgumentException(INVALID_ELEMENT_MESSAGE);
+        }
+    }
+
+    private static void isBlank(String str) {
+        if (str.isBlank()) {
+            throw new IllegalArgumentException(EMPTY_INPUT_MESSAGE);
+        }
+    }
```

Comment:
메서드명이 `isBlank()`면 54번 라인에 작성하신 `isBlank()` 처럼 작동할 것을 예상하게 돼요!
이런 이름은 어떠신가요?
```suggestion
    private static void validateNotBlank(String str) {
        if (str.isBlank()) {
            throw new IllegalArgumentException(EMPTY_INPUT_MESSAGE);
        }
    }
```